test/libsolidity/smtCheckerTests/abi/abi_encode_call_simple.sol

```diff
@@ -0,0 +1,26 @@
+contract C {
+	function callMeMaybe(uint a, uint b, uint[] memory c) external {}
+
+	function abiEncodeSimple(uint x, uint y, uint z, uint[] memory a, uint[] memory b) public view {
+		require(x == y);
+		bytes memory b1 = abi.encodeCall(this.callMeMaybe, (x, z, a));
+		bytes memory b2 = abi.encodeCall(this.callMeMaybe, (y, z, a));
+		assert(b1.length == b2.length);
```

Comment:
This assertion should hold because `x == y`. *This* might then be because it's using the tuple of arguments and not the arguments

Reply:
When I test locally without the array being part of `callMeMaybe`, the solver finds a counterexample for it (that's why I wrote the above).